### PR TITLE
KIP-1 Harmonize Base Rate

### PR DESCRIPTION
**Pull request by arthurka-o**

### CODE REVIEW

File: packages/contracts/test/BorrowerOperationsTest.js
Comment (udosson) [inline packages/contracts/test/BorrowerOperationsTest.js:1]: @arthurka-o - can you add the following tests, please:
(1) redemption should increase the borrowing rate of assets one and two
(2) after redemption, the borrowing rate of assets one and two should decrease over time
(3) Check that we can set different `REDEMPTION_FEE_FLOOR` and `BORROWING_FEE_FLOOR` and check its implications after a redemptions

Comment (udosson) [reply packages/contracts/test/BorrowerOperationsTest.js:1]: Please decide in which test file the tests make the most sense. thx

**Review by udosson — COMMENTED** [review]
@arthurka-o - the PR already looks very good to me. Please add the tests I mentioned in the comment.

@asimpk - can you please have a closer look at the SDK-related changes and review them?

**Comment by asimpk** [comment]
> @arthurka-o - the PR already looks very good to me. Please add the tests I mentioned in the comment.
> 
> @asimpk - can you please have a closer look at the SDK-related changes and review them?

Hi @udosson , the only change I saw it in the readable call, It doesn't affect the state but I'll confirm it from @arthurka-o

**Review by udosson — APPROVED** [review]
@arthurka-o - LGTM!!!! Well done

**Review by udosson — APPROVED** [review]
Thanks for adding the last test Arthur. Please, feel free to merge

**Pull request merged**